apps/artwork_purchase/index.styl

```diff
@@ -86,7 +87,7 @@ a#purchase-page-header-title
       // margin 30px
       garamond-size('l-body')
       clearfix()
-      &:not(.ap-summary-price) .ap-summary__right
+      &:not(.ap-summary__price) .ap-summary__right
```

Comment:
The reason it wasn't black was because I changed the class names at some point and made them all __ underscores instead of dashes, but I forgot to update it here. So when this :not had a selector that didn't actually match the div containing the price, the price was being styled black according to this rule (becasue it is also .ap-summary__right)